driver/ppm.h

```diff
@@ -106,4 +106,12 @@ extern void ppm_syscall_get_arguments(struct task_struct *task, struct pt_regs *
 #define SECOND_IN_NS 1000000000
 #define USECOND_IN_NS 1000
 
+// used in main.c, ppm_events.c and ppm_fillers.c so include it just once here
```

Comment:
thank you for this! Can we remove the `#include "ppm_syscall.h"` from the aforementioned files?

Reply:
Possibly. Will give it a try.